### PR TITLE
Add linting

### LINKED ISSUE #57: Add eslint
Add [eslint](http://eslint.org/) to the project. My only preferences are for a style with:

* no semicolons
* single quotes

Note: This should only require adding a `.eslintrc` file.  We can add a `package.json` so that people can automatically install our configuration in another diff.

### PR DESCRIPTION
**Pull request by CoreyAR**
Closes #57

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+  "name": "nashviva",
+  "version": "1.0.0",
+  "description": "A digital pocketknife for life in Nashville",
+  "scripts": {
+    "lint": "standard --verbose | snazzy",
+    "fixcode": "standard --fix",
+    "precommit": "if [ -d 'ignite-base' ]; then cd ignite-base; fi; npm run lint"
```
Comment (mshenfield) [inline package.json]: Remove

Comment (CoreyAR) [inline package.json]: Ahh yeah, I meant to set that up with [husky](https://www.npmjs.com/package/husky).

Comment (mshenfield) [inline package.json]: TIL - that's fantastic.  What's `ignite-base` in this though?

Comment (CoreyAR) [inline package.json]: Whoops. I learned about this setup from the Ignite react-native starter. I think I have just been carrying that extra code in the project I copied it from. Removed.

**Pull request merged**